README.md

```diff
@@ -87,6 +87,25 @@ You can then update your test script in `package.json` to use babel:
 "test": "node -r @babel/register -r @babel/polyfill source/test"
 ```
 
+If you structure your folders by type like this:
+
+```bash
+├──todos
+│  ├── component
+│  ├── reducer
+│  └── test
+└──user
+   ├── component
+   ├── reducer
+   └── test
+```
+
+Update your test script to find all files with your custom ending:
+
+```json
+"test": "node -r @babel/register -r @babel/polyfill ./node_modules/.bin/riteway 'src/**/*.test.js' | tap-color",
```

Comment:
You can leave the path out and just call `riteway` and pass a blob.

Reply:
Hmm, for me (working with Create React App) both 

```json
"test": "node -r @babel/register -r @babel/polyfill riteway src/**/*.test.js | tap-color",
```

and

```json
"test": "node -r @babel/register -r @babel/polyfill riteway 'src/**/*.test.js' | tap-color",
```
won't work. And `"test": "riteway src/**/*.test.js | tap-color",` throws the error that 'import' is not defined (because babel).

Reply:
@ericelliott Any updates? What do you mean with you can leave out the path? 😊

Reply:
Did you try `riteway -r  @babel/register -r @babel/polyfill 'src/**/*.test.js' | tap-color`?

Reply:
@ericelliott No I didn't. Now I did, it works (thank you!) and I updated the PR.